docs/src/app/components/pages/components/dialog.jsx

```diff
@@ -76,19 +76,13 @@ export default class DialogPage extends React.Component {
             desc: 'Force the user to use one of the actions in the dialog. Clicking outside the dialog will not dismiss the dialog.',
           },
           {
-            name: 'openImmediately',
```

Comment:
We should ideally (for the v0.13.2 release). Do you mind if it's not too much work?